### DISCUSSION TITLE
I wanna get PI~

### DISCUSSION THREAD

**Discussion by HeW3** [3b1b/manim#1842] [Q&A]
I'm new to manim community
But...
The PI in the 3b1b's video is so CUTE💕~~💕~💕~~💕~~~~~~~~
Where can I get all of them(the whole family)💕💕💕💕~~~~~~~~~
![image](https://user-images.githubusercontent.com/97308223/179792958-3b169622-03d3-4f7c-ab72-dde36e5a84ba.png)
@3b1b 
PI~~PI~~PI~~~PI~PI~~~PI~~~~~PI~~PI~PI~~~PI~PI~PI~~~

**Comment by CaftBotti** [comment]
See this: `https://github.com/CaftBotti/manim_pi_creatures`
